app/controllers/activities_controller.rb

```diff
@@ -7,12 +7,31 @@ def index
 
   def create
     @activity = MessageActivity.create(message_params)
-    PusherActivityNotificationJob.perform_later(appointment.guider_id, @activity.id)
     respond_to do |format|
       format.js { render @activity }
     end
   end
 
+  def update_resolve
```

Comment:
Sure, I was following the pattern I saw elsewhere, so I'll update this and the other instance as well.